From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: bergschneider <magiclake@gmx.de>
Date: Fri, 26 May 2023 12:59:18 +0200
Subject: [PATCH] add body rotation to api


diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
index 9193288438671409cc3cf92033ef7cb60d798b69..a3b4473dd2b8093c9c583dd2a523f1b46f78f2b3 100644
--- a/src/main/java/org/bukkit/entity/Player.java
+++ b/src/main/java/org/bukkit/entity/Player.java
@@ -3397,6 +3397,40 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
      */
     void sendDeathScreen(@NotNull net.kyori.adventure.text.Component message);
 
+
+    // Pegasus start
+
+    /**
+     * Returns the player's body rotation.
+     *
+     * @return body rotation
+     */
+    float getBodyRotation();
+
+    /**
+     * Sets the player's body rotation.
+     *
+     * @param bodyRotation body rotation
+     * @see #getBodyRotation()
+     */
+    void setBodyRotation(float bodyRotation);
+
+    /**
+     * Returns the player's body rotation.
+     *
+     * @return body rotation
+     */
+    float getBodyRotationO();
+
+    /**
+     * Sets the player's body rotation.
+     *
+     * @param bodyRotation body rotation
+     * @see #getBodyRotation()
+     */
+    void setBodyRotationO(float bodyRotationO);
+
+    // Pegasus end
     /**
      * Sends a player the death screen with a specified death message,
      * along with the entity that caused the death.
