From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: bergschneider <magiclake@gmx.de>
Date: Fri, 26 May 2023 12:59:18 +0200
Subject: [PATCH] add body rotation to api


diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
index dc437885404ae147a06cac653e519a4674a9a951..f261635dacbde140456b593bcea9d6af28473767 100644
--- a/src/main/java/org/bukkit/entity/Player.java
+++ b/src/main/java/org/bukkit/entity/Player.java
@@ -3203,4 +3203,38 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
      */
     void sendDeathScreen(@NotNull net.kyori.adventure.text.Component message, @Nullable Entity killer);
     // Purpur end
+
+    // Pegasus start
+
+    /**
+     * Returns the player's body rotation.
+     *
+     * @return body rotation
+     */
+    float getBodyRotation();
+
+    /**
+     * Sets the player's body rotation.
+     *
+     * @param bodyRotation body rotation
+     * @see #getBodyRotation()
+     */
+    void setBodyRotation(float bodyRotation);
+
+    /**
+     * Returns the player's body rotation.
+     *
+     * @return body rotation
+     */
+    float getBodyRotationO();
+
+    /**
+     * Sets the player's body rotation.
+     *
+     * @param bodyRotation body rotation
+     * @see #getBodyRotation()
+     */
+    void setBodyRotationO(float bodyRotationO);
+
+    // Pegasus end
 }
