From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: bergschneider <magiclake@gmx.de>
Date: Fri, 26 May 2023 12:58:51 +0200
Subject: [PATCH] add body rotation to api


diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
index 018cf3cd424ac85e8d84179bc05ab241f0a00827..750bf990d5135edfc245d856f1c48016cec7dafc 100644
--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
@@ -3377,4 +3377,27 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
         this.getHandle().connection.send(new net.minecraft.network.protocol.game.ClientboundPlayerCombatKillPacket(getEntityId(), io.papermc.paper.adventure.PaperAdventure.asVanilla(message)));
     }
     // Purpur end
+
+
+    // Pegasus start
+    @Override
+    public float getBodyRotation() {
+        return this.getHandle().yBodyRot;
+    }
+
+    @Override
+    public void setBodyRotation(float bodyRotation) {
+        this.getHandle().yBodyRot = bodyRotation;
+    }
+
+    @Override
+    public float getBodyRotationO() {
+        return this.getHandle().yBodyRotO;
+    }
+
+    @Override
+    public void setBodyRotationO(float bodyRotation) {
+        this.getHandle().yBodyRotO = bodyRotation;
+    }
+    // Pegasus end
 }
